src/main/kotlin/com/mineinabyss/staminaclimb/nms/Tags.kt

```diff
@@ -9,7 +9,7 @@ import net.minecraft.network.protocol.game.ClientboundUpdateTagsPacket
 import net.minecraft.resources.ResourceLocation
 import net.minecraft.tags.BlockTags
 import net.minecraft.tags.TagNetworkSerialization.NetworkPayload
-import org.bukkit.craftbukkit.v1_19_R3.entity.CraftPlayer
+import org.bukkit.craftbukkit.v1_20_R1.entity.CraftPlayer
```

Comment:
These changes have already been made on master, maybe you need to rebase something?